scripts/zones/East_Ronfaure/npcs/Geomantic_Reservoir.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- Area: East Ronfaure
+-- NPC: Geomantic Reservoir
+-- Unlocks Geo-Poison
+-- !pos 378.875 -39.304 58.313
+-----------------------------------
+local ID = require("scripts/zones/East_Ronfaure/IDs")
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+end
+
+entity.onTrigger = function(player, npc)
+    if player:hasSpell(769) then -- Player has Indi-Poison
+        player:startEvent(15000, 277, 3, 380619, -39102, 53620, 163, 539943, 0)
```

Comment:
Most of these args are probably junk and can be omitted, you could try `15000`, `15000, 277` or `15000, 277, 3` and see if it still appears correctly.